lib/gui/lvgl/Kconfig.graphical

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2018-2019 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+# Copyright (c) 2020 Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+menu "Graphical settings"
+
+config LVGL_DISPLAY_DEV_NAME
+	string "Display device name"
+	default "DISPLAY"
+	help
+	  Name of the display device to use for rendering.
+
+config LVGL_HOR_RES
+	int "Horizontal Screen Resolution"
+	default 480
```

Comment:
From your comment on the sample code should the default HOR_RES not be 640 and VER_RES 480?

Reply:
@vanwinkeljan I said it wrong, it's 480x320

Reply:
Are the new defaults for resolution, refresh period, etc related to the upgrade to v7.0.2? If not, they should be changed in a separate commit and the motivation for the change explained in the commit message.

Reply:
As already said, I have adjusted defaults to match v7 defaults, which I think it’s the reasonable thing to do. If this is not wanted let me know.

Reply:
I would also keep the the LVGL defaults as it is a major version up step

Reply:
> As already said, I have adjusted defaults to match v7 defaults, which I think it’s the reasonable thing to do. If this is not wanted let me know.

"I have adjusted defaults to match v7 defaults" but why, what is the technical background for that? Why are higher values necessary?

Reply:
Let me ask the other way around, what is the benefit of Zephyr having different defaults than LVGL?

Reply:
There is no need for benefit, there is no need to change defaults we have and keeping them would not force to change config file for samples. Last time: keep the defaults or describe why changes are necessary with a technical background not just because you feel better with the changes.

Reply:
The technical background: it makes sense to use defaults from LVGL, which is the part that has the expertise and it is likely to know which ones work better. Zephyr should be as transparent as possible.

Reply:
@jfischer-phytec-iot ?

> okay, I have expertise too and say 320 works better.